### PR TITLE
added yarn functionality and removed npm

### LINKED ISSUE #2: Setup the backend repository
- Setup the Backend folder with yarn and Mongodb
- Create the initial file structure

### PR DESCRIPTION
**Pull request by 17himanshu**
Issue Resolved : #2

### CODE REVIEW

**Comment by 17himanshu** [comment]
@ravinder-chadha Hi sir, I have removed npm and reinitialized setup with yarn and installed Express and Mongoose

File: backend/.gitignore [hunk 1]
```diff
@@ -0,0 +1,329 @@
+<<<<<<< HEAD
+# -------- Ignoring the files and directories that are not required to be pushed to the repository --------
+
+
+
+# For Node.js projects, the following files and directories are not required to be pushed to the repository
+=======
+>>>>>>> 998fe05 (removed npm added yarn functionality)
```
Comment (ravinder-chadha) [inline backend/.gitignore]: Please resolve the merge conflicts here, I don't think you need a `.gitignore` inside the backend folder, as it is already present in the root directory.

File: backend/.gitignore [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+#  option (not recommended) you can uncomment the following to ignore the entire idea folder.
+#.idea/
+# End of Python project specific files and directories
+
+
+
+# -------- End of ignoring the files and directories that are not required to be pushed to the repository --------
+=======
+.pnp.*
+>>>>>>> 998fe05 (removed npm added yarn functionality)
```
Comment (ravinder-chadha) [inline backend/.gitignore]: Same here

**Comment by ravinder-chadha** [comment]
Please create the rest of the folder and file structures as well. So, that you are good to start with the next issue.

File: backend/.gitignore (hunk 1 above)
Comment (17himanshu) [reply backend/.gitignore]: Sure Sir

**Pull request closed without merging**